### ISSUE TITLE
Set up Quickbooks

### ISSUE THREAD

**Issue by patcon** [hyphacoop/organizing#154]
<sup>_This initial comment is collaborative and open to modification by all._</sup>
## Task Summary

🎟️ **Re-ticketed from:** #  
🗣 **Loomio:** N/A  
📅 **Due date:** N/A  
🎯 **Success criteria:** We are ready to do our accounting with quickbooks, and feel confident we're doing it correctly.   

...

cc: @hyphacoop/finance-wg 

## To Do

- [x] sign up for quickbooks -- @benhylau 
- [x] connect to bank account -- @benhylau 
- [x] categorize prior transactions -- @YurkoWasHere 
- meta actions
  - [x] add completed tasks to this issue
  - [x] spec out and decide remaining tasks in this issue
- [x] review completed actions in finances wg meeting (suggested by yurko)
  - [x] review categorizations of entries
  - [x] review accounts created
- research tasks
  - [x] investigate how to handle costs before incorporation
    - [x] summarize [Turbotax: Deductible Business Start-Up Costs](https://turbotax.intuit.ca/tips/deductible-business-start-up-costs-6184)
    - [x] summarize [CRA: Business Transactions Prior to Incorporation](https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/it454/archived-business-transactions-prior-incorporation.html)
  - [x] investigate how to handle shareholder loans
    - [x] summarize [Shareholder Loan: Understand it and Avoid Trouble with the CRA](https://www.avalonaccounting.ca/blog/shareholder-loan). https://github.com/hyphacoop/organizing/issues/154#issuecomment-561358717
    - [x] add shareholder loans to QB. [howto.](https://quickbooks.intuit.com/learn-support/en-us/journal-entries/how-to-record-a-company-loan-from-a-company-officer-or-owner/00/185614) -- @YurkoWasHere 
- [x] invoices, re: https://github.com/hyphacoop/organizing/issues/120#issuecomment-557923816
  - [x] create international invoice template. [done](https://c5.qbo.intuit.com/app/customizeforms?templatePrefId=500000000000273421&formType=INVOICE) :lock:
    - ❗️ payment/footer can be per-template customized, but customer greeting is global.
  - [x] create shortlinks for tax references https://github.com/hyphacoop/shortlinks/pull/16
    - https://link.hypha.coop/ref-taxes-export-services-ip
    - https://link.hypha.coop/ref-taxes-hst-ecommerce
  - [x] add tax references to international invoice
  - [x] add Ontario business number to master invoice template ([ref](https://medium.com/inside-formstack/working-remotely-for-an-american-company-as-a-canadian-e96296eea477))
  - [x] add thank-you
- work with our accountant
  - [x] have accountant/bookkeeper give our system a once-over
  - [x] extract tasks from conversation
- [ ] set up sales tax. [quickbooks link.](https://c5.qbo.intuit.com/app/tax/home) [screenshot](https://i.imgur.com/bkUdres.png) (deferred to HST task)
  - [x] get zero-rating showing as 0% on international invoice templates
- [x] add default payment terms to invoice templates: NET 15?
- [x] ~~create pro forma invoice template~~ using invoice instead
- [ ] ...

**Comment by patcon** [comment]
Shared by @YurkoWasHere in chat: https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/it454/archived-business-transactions-prior-incorporation.html

**Comment by YurkoWasHere** [comment]
Review new accounts created

- Loan from Owners
- Organizational Costs
- Computer and Internet Expenses

Decide what to do with Interest prior to aug1/2019

**Comment by YurkoWasHere** [comment]
HST for NonProfits:

https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/rc4081/gst-hst-information-non-profit-organizations.html

**Comment by benhylau** [comment]
Thanks @YurkoWasHere the "non-profit organization promoting the arts in your local community in Nova Scotia" example is very helpful. Basically we'd charge HST for everything except mission-aligned initiatives (e.g. workshops), and we are eligible to claim _Input Tax Credit (ITC)_ on expenditures not relating to already exempt activities.

## International Clients

For international clients, it seems we do not need to collect HST on most of our digitally-offered services (e.g. software development, devops, design) because it is classified as _exported goods and services_, which is zero-rated. This means we can also collect ITC on related expenditures, since _zero-rated_ is not the same as _exempt_. See: [Intangible personal property](https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/b-090/gst-hst-electronic-commerce.html#P22_9331) and [Excise Tax Act section 10 of Part V of Schedule VI](https://laws-lois.justice.gc.ca/eng/acts/E-15/page-179.html#h-201391). The latter text reproduced below:

```
10 A supply of an invention, patent, trade secret, trademark, trade-name, copyright, industrial design or other intellectual property or any right, licence or privilege to use any such property, where the recipient is a non-resident person who is not registered under Subdivision D of Division V of Part IX of the Act at the time the supply is made.

10.1 A supply of intangible personal property made to a non-resident person who is not registered under Subdivision D of Division V of Part IX of the Act at the time the supply is made, but not including

    (a) a supply made to an individual unless the individual is outside Canada at that time;

    (b) a supply of intangible personal property that relates to

        (i) real property situated in Canada,

        (ii) tangible personal property ordinarily situated in Canada, or

        (iii) a service the supply of which is made in Canada and is not a zero-rated supply described by any section of this Part or Part VII or IX;

    (c) a supply that is the making available of a telecommunications facility that is intangible personal property for use in providing a service described in paragraph (a) of the definition telecommunication service in subsection 123(1) of the Act;

    (d) a supply of intangible personal property that may only be used in Canada; or

    (e) a prescribed supply.
```

My read is that these activities are intellectual property exports made in Canada, which are zero-rated for the purpose of HST collection.

More accessible but unofficial references below:

- http://vhaccounting.ca/2019/03/08/gst-hst-on-remote-services/
- https://www.reddit.com/r/PersonalFinanceCanada/comments/5uwjkw/do_i_have_to_charge_my_american_clients_gstpst/

**Comment by patcon** [comment]
Link shared by yurko in chat: https://www.avalonaccounting.ca/blog/shareholder-loan

Migrated @YurkoWasHere's suggested review tasks [from chat](https://matrix.to/#/!JloFJtoONurMdgcZCa:tomesh.net/$15742949532285uWXea:tomesh.net?via=tomesh.net&via=elon.li)

**Comment by patcon** [comment]
Added a todo for setting up taxes. Is accrual vs cash and filing frequency a big decision, or something we can decide on our own as finance WG?

**Comment by patcon** [comment]
## Tentative to dos

- [ ] add receipts to quickbooks

**Comment by darkdrgn2k** [comment]
> 
> EDIT: I later found: https://turbotax.intuit.ca/tips/deductible-business-start-up-costs-6184
> 

Usually Small Business != Corporation  

Although I'm not 100%,  i don't think this applies to corporations

**Comment by darkdrgn2k** [comment]
>     * [ ]  regroup interest charges to incorp date as initial loan

If interested charges are "initial loan" then who gets that money back?

:thought_balloon: dont think this is the right catagory

**Comment by benhylau** [comment]
## Multi-currency invoicing

Turns our QB support for multi-currency (which include invoicing) is only available in [Essentials and above](https://quickbooks.intuit.com/ca/pricing/), so we don't have it unless we upgrade. Even then, [the feature seems quite restrictive](https://quickbooks.intuit.com/learn-support/en-ca/sales-and-invoicing/can-i-send-invoice-in-canadian-dollars-and-us-dollars-i-bill-in/00/261151). For example, if we need to bill one customer in different currencies for different projects, we basically need to create different customers (e.g. ABC Tech, ABC Tech (USD)) because the currency is associated with customer. This probably is okay for our current purpose, but I feel committing ourselves to pay more money per month for something quite restrictive may be a bad idea, although with Essentials we do get multi-user, which may be useful.

An alternative is we make only CAD receipts, but put the native currency info into the description:

| Description | QTY | RATE | AMOUNT |
|:------------|:-----|:------|:----------|
| Software development for Aug 2019 (10.00 @ USD 50.00 = 500.00) | 10 | 66.5 | 665 |

Include a line in the end that says:

>Please pay CAD 665.00 or USD 500.00

Then include TransferWise USD payment info and Desjardins/TransferWise CAD payment info in the footer.

@hyphacoop/finance-wg thoughts?

**Comment by YurkoWasHere** [comment]
Yep sounds about right :(

Trying to do this in multi currency without proper support from the accounting software will create chaos. especially when it comes to HST. Remember HST is "paid" on when we invoice NOT on when we get paid.

I think your way @benhylau is the best way forward currently, where the books themselves just deal in CAD.

I think if we send the invoice as a **estimate** of how much it would be after conversion. (ie 665.00)
Then once the payment goes through we update the invoice (with an additional line) to indicate the correct amount after conversions and that way we balance the books.

Question: we are not **keeping** the money in transfer wise right?

**Comment by darkdrgn2k** [comment]
```
* [ ]  regroup interest charges to incorp date as initial loan
```

If interested charges are "initial loan" then who gets that money back?

💭 dont think this is the right catagory

**Comment by YurkoWasHere** [comment]
```
* [ ]  regroup interest charges to incorp date as initial loan
```

If interested charges are "initial loan" then who gets that money back?

💭 don't think this is the right category

**Comment by YurkoWasHere** [comment]
> I later found: https://turbotax.intuit.ca/tips/deductible-business-start-up-costs-6184

Usually Small Business != Corporation
Although I'm not 100%, i don't think this applies to corporations

**Comment by patcon** [comment]
> Although I'm not 100%, i don't think this applies to corporations

The example they use is a brick and mortar storefront, which I believe would have to be an 
corporation, no? Also, the CRA links at bottom defines a "business" very generally, and direct readers to consider a corporation. Though open to idea that what you're saying is correct, I feel they'd make the distinction more clear if the article _didn't_ apply to corporations...?

**Comment by patcon** [comment]
Thanks Ben! I'm easy on this, since y'all working on Aether know best how this client prefers.

Having said that, here are my thoughts for your consideration:

- **Quickbooks**
  - :heart: Being without multiuser hasn't felt inconvenient. Haven't noticed any pain points.
  - :heart: Not opposed to upgrading QB now or later.
  - :heart: I feel that if we wish to explore Odoo in the future, multi-currency might surface more edge-cases and complication.
  - :heart: If we can make multicurrency work for us within **Transferwise**, I'd feel slightly more favourable to that.
- :mag: Our only US client already uses **Transferwise**.
  - :mag: TransferWise has an ["amount lock" feature](https://transferwise.com/help/11/getting-started/2932114/amount-lock-transfers-specific-amount-transfers)
    - :heart: This seems to make sending in foreign (i.e. CAD) rates easy for clients.
  - :heart: If we have a future US client for which using Transferwise would be inconvenient, then we could deal with that when it comes up.
- :bulb: We could keep the books only in CAD.
  - :bulb: We could helpfully offer an estimated USD approximation as note on invoice.
  - :bulb: We could explicitly direct client to use TransferWise amount lock feature to send the billed amount in CAD.

**Comment by patcon** [comment]
> Question: we are not keeping the money in transfer wise right?

Not sure if it answers fully, but there's something in [Finance section of our handbook](https://handbook.hypha.coop/finance.html#financial-accounts) about it:

> Our TransferWise account does not require multiple signatures, and the Finance WG is responsible for transfering money into our main Desjardins account **when accumulated funds exceed a CAD 10,000 threshold.**

**Comment by YurkoWasHere** [comment]
> @YurkoWasHere Not sure if it answers fully, but there's something in Finance section of our handbook about it

Yes that does answer the question. Thanks.

However without being able to record money in a USD account (transfer wise being a new account) in QB the process may need to be revised.

**Comment by patcon** [comment]
> If interested charges are "initial loan" then who gets that money back?

Could we just divide the interest amount between everyone, and add it to the amount we each supposedly initially loaned?

**Comment by patcon** [comment]
> @YurkoWasHere: However without being able to record money in a USD account (transfer wise being a new account) in QB the process may need to be revised.

I believe if we used the amount lock feature as per my suggestion, that brings the money directly into our CAD account on TransferWise. So we'd have nothing coming in as USD. Correct me if I'm wrong Ben

**Comment by YurkoWasHere** [comment]
> 
> 
> > @YurkoWasHere: However without being able to record money in a USD account (transfer wise being a new account) in QB the process may need to be revised.
> 
> I believe if we used the amount lock feature as per my suggestion, that brings the money directly into our CAD account on TransferWise. So we'd have nothing coming in as USD. Correct me if I'm wrong Ben. (But also, yes, mentioning USD seems a good idea!)

As long as the funds are stored in CAD this should work! 

thanks!

**Comment by patcon** [comment]
## Summary: [Shareholder Loan: Understand it and Avoid Trouble with the CRA](https://www.avalonaccounting.ca/blog/shareholder-loan)

- shareholder loan
  - commonly happening "under the curtains" by bookkeepers
  - any funds borrowed _from_ or lent _to_ an org by owners/officers
  - aka "due _from_ shareholder" when owner _borrows_
    - type: cash withdrawal
    - type: personal (non-business) item purchase. e.g., when accidental.
      - "recorded to Paul’s shareholder loan account"
  - aka "due _to_ shareholder" when owner _lends_
    - type: owner cash contribution. 👈 our loans
    - type: business expense with personal funds. e.g., out for beer with client.
      - "recorded to Joe’s shareholder loan account"
- shareholder loan balance = running total of all shareholder loan transactions
  - appears on your balance sheet as either an _asset_ or a _liability_. [screencap](https://images.squarespace-cdn.com/content/v1/531500eee4b06a95ceb84c1e/1550092504806-0AA14RC2O5ZEO9RKGOFF/ke17ZwdGBToddI8pDm48kNnJIuhQlzEQPo_7_Lg0HaRZw-zPPgdn4jUwVcJE1ZvWQUxwkmyExglNqGp0IvTJZamWLI2zvYWH8K3-s_4yszcp2ryTI0HqTOaaUohrI8PIII0_NSAfNztsvIJtibKYRuI3H4JPFGnA4uawSjG5HGEKMshLAGzx4R3EDFOm1kBS/Shareholder+Loan?format=750w)
- "CRA tax problems can arise when companies make loans to shareholders over a period of more than a year."
  - "might consider that loan to be personal income"
    - double taxation possible
    - ❓ inverse true for loan from shareholder? might loan be considered corporate income?
- important that the loan is properly **documented**
  - terms of repayment in a **written agreement or corporate resolution**
- 3 ways to avoid double-taxation (framed as loan to owner; **we a _from_ owner**)
  - repay the loan. permanently before end of the year.
  - take the cash as salary/wage
  - take the cash as dividend

**Comment by patcon** [comment]
Started adding bookkeeper questions here: https://github.com/hyphacoop/organizing/issues/122

**Comment by patcon** [comment]
Finds related to pro forma invoices that @YurkoWasHere mentioned [in chat](https://matrix.to/#/!JloFJtoONurMdgcZCa:tomesh.net/$15754752272802RVxig:tomesh.net?via=tomesh.net&via=elon.li):

> USD and Invoices: She had a novel solution for this issue. In the past when dealing with multi currency she would do a PRO FORMA invoice instead of an invoice.
>
> This way the invoice is issued, the payment is made, then we can adjust the proformal invoice with the correct canadian amount and issue a final invoice with all the information updated and correct

Tutorial for proforma invoices in Quickbooks as "estimates": https://www.arbitrue.com/blog/how-to-issue-estimates-and-pro-forma-invoices-in-quickbooks/

> In QuickBooks, estimates have a similar list of default elements to invoices with some differences.
> - Due dates and payment terms, such as Net 30, are not available

https://quickbooks.intuit.com/ca/resources/invoicing/5-things-you-need-to-know-about-invoicing/

> Not stating the terms of payment on your invoice is similar to handing someone a blank cheque;

https://debitoor.com/blog/5-things-to-know-about-proforma-invoices

> Because they don’t meet the requirements of a true invoice, proforma invoices **don’t carry the same legal weight**, which means that:
> - Customers aren’t required to pay the amount listed on a proforma invoice
> - The total amount due shouldn’t be recorded under a customer’s accounts payable or a supplier’s accounts receivable
> - You can’t use proforma invoices to reclaim VAT. 👈 "don't need to consider HST yet?"

**tl;dr** - Seems that pro forma means "this is not yet due" in legalese. So despite advice, thinking we should prob stick with our original strategy of "issue prelim invoice, and change after".

**Comment by dcwalk** [comment]
@patcon We aren't a shareholder corporation, so while those concerns flagged by CRA make sense, we aren't in the same position in terms of a relationship of owners to shareholders by making a cash loan.

That might not make a difference with respect to the "worst" implication for us: double income, but the amount is _so low_ I'm not sure an extra $150 being reported as income is going to significantly impact anyones return...?

**Comment by benhylau** [comment]
re: shareholder loan +1 to what @dcwalk said re: amount being low. I think further researching it and having everyone to read it is not a good use of our time. My proposal is we book it the way @YurkoWasHere has done, then let our bookkeeper/accountant decide whether adjustments need to be made when it comes time for filing.

**Comment by benhylau** [comment]
@YurkoWasHere idk what is the difference between these two from perspective of bookkeeping:

1. Issue invoice, receive payment, adjust invoice
2. Issue pro forma invoice, receive payment, create invoice

I know @patcon is concerned that pro forma isn't legally binding. I am personally not super concerned with current clients not paying, but I wonder if it may confuse some finance departments of larger future clients. Like if a big org needs to pay us, and they forward the pro forma to their finance dep, will they be like "huh? can you send me the final invoice?"

I have paid pro formas before, but they are usually to purchase stuff, which means the invoices were never "due", as they wouldn't ship me the thing until it's paid :)

I am happy to start with either 1 and 2, then if the method we use create confusion / accounting nightmare, we switch to the other. I think they are pretty close that it doesn't have to be a big discussion, and would propose to let the person in charge of the books set up, aka. @YurkoWasHere, to make this decision. @patcon are you okay with that?

**Comment by YurkoWasHere** [comment]
> Issue invoice, receive payment, adjust invoice

Looks shady from book keeping practice. I think usually adjustments like that you issue a NEW invoice or issue a credit something-or-another to adjust out the balance for the client.

This may be a good question to the book keeper during the review.

**Comment by patcon** [comment]
Rodica said that the way we're doing shareholder loans looks fine.

**Comment by patcon** [comment]
https://agencymanagementinstitute.com/why-do-agencies-use-the-accrual-method-of-accounting/
